### PR TITLE
Ion Auth 3: Documentation update

### PR DESCRIPTION
**Pull request by Indigo744**
#1195 

Here is some update for the docs and the migration support.

### CODE REVIEW

**Comment by Indigo744** [comment]
This PR should be merge at the end. I'll update it with relevant information as needed.

File: INSTALLING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+```shell
+CI                          # → Root Directory
+└── application/
+    ├── config/
+    │   └── ion_auth.php
+    ├── controllers/
+    │   └── Auth.php
+    ├── libraries
+    │   ├── Bcrypt.php
```
Comment (benedmunds) [inline INSTALLING.md]: this can be removed

File: INSTALLING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+CI                          # → Root Directory
+└── application/
+    ├── controllers/
+    │   └── Auth.php
+    ├── third_party/
+    │   └── ion_auth/
+    │       ├── config/
+    │       │   └── ion_auth.php
+    │       ├── libraries
+    │       │    ├── Bcrypt.php
```
Comment (benedmunds) [inline INSTALLING.md]: This can be removed

File: README.md [hunk 3]
```diff
@@ -6,9 +6,11 @@ Redux Auth 2 had a lot of potential.  It's lightweight, simple, and clean,
 but had a ton of bugs and was missing some key features.  So we refactored
 the code and added new features.
 
-This version drops any backwards compatibility and makes things even more
-awesome than you could expect.
+Ion Auth 2 dropped any backwards compatibility and made things more
```
Comment (benedmunds) [inline README.md:9]: "Ion Auth 3"

Comment (Indigo744) [reply README.md:9]: Well actually this sentence is really about Ion Auth 2. The next sentence is about Ion Auth 3.
But I don't know, it feel kinda weird. You may want to phrase it the way you prefer ;-)

**Comment by Indigo744** [comment]
I've updated the user guide.

**Comment by benedmunds** [comment]
Looks good, merging as is and I’ll circle back to tune some of the wording in the readme later.

**Pull request merged**